src/storage/CMakeLists.txt

```diff
@@ -4,6 +4,8 @@ add_subdirectory(remote)
 set(
   sources
   Storage.cpp
+  Storage.hpp
+  types.hpp
```

Comment:
Would it make sense to glob these as well?

Reply:
Sure.

Reply:
Done